charts/refinery/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: refinery
 description: Chart to deploy Honeycomb Refinery
 type: application
-version: 1.18.0
+version: 1.19.0
```

Comment:
Newb question: It's interesting to me that the version of the chart is different than the version of refinery. Do we ever try to keep those the same?

Reply:
A helm chart has an independent release and version for the thing it manages.  The main reason for this is because the chart might have features/bug fixes/breaking changes that are independent of the underlying service that need to be release to the helm chart users.  For example, if we needed to make a change to the chart to handle some new k8s feature, we could release that without needing to release Refinery.